### PR TITLE
[WIP]: Add News Page

### PR DESCRIPTION
**Pull request by vpranavkrishna**

### CODE REVIEW

File: lib/pages/news_page.dart [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    // color: ,
+    decoration: BoxDecoration(
+        color: baseColor, borderRadius: BorderRadius.circular(10)),
+
+    child: news_list(),
+  );
+}
+
+// ignore: non_constant_identifier_names
+BlocBuilder<MarketEventBloc, MarketEventState> news_list() {
```
Comment (Sudhindra3000) [inline lib/pages/news_page.dart:51]: Make the return type for all helper methods as `Widget`

File: lib/pages/news_page.dart [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      return const Center(
+        child: CircularProgressIndicator(
+          color: secondaryColor,
+        ),
+      );
+    }
+  });
+}
+
+Container newsItem(String headline, String text, String imagePath) {
```
Comment (Sudhindra3000) [inline lib/pages/news_page.dart:81]: Make a separate widget for list view items

**Review by Sudhindra3000 — CHANGES_REQUESTED** [review]

File: lib/pages/news_page.dart [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import 'package:dalal_street_client/main.dart';
+import 'package:dalal_street_client/proto_build/models/MarketEvent.pb.dart';
+import 'package:dalal_street_client/theme/colors.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../blocs/market_event/market_event_bloc.dart';
+
+// ignore: camel_case_types
+class News_Page extends StatefulWidget {
```
Comment (Sudhindra3000) [inline lib/pages/news_page.dart:10]: Make it camel case. Remove underscore

**Review by Sudhindra3000 — CHANGES_REQUESTED** [review]

File: lib/pages/news_page.dart (hunk 3 above)
Comment (Sudhindra3000) [reply lib/pages/news_page.dart:10]: For all other classes and methods as well

**Comment by siva2204** [comment]
- `getMarketEvents` rpc returns latest 10 news or the count number (which is part of request) only.
- there are other old news to fetch if  [more_exists](https://github.com/delta/dalal-street-proto/blob/master/actions/GetMarketEvents.proto#L23) field in response is true.

may we can have a button to load old news in ui

**Pull request closed without merging**